src/cpp/session/modules/SessionHelp.R

```diff
@@ -406,6 +406,31 @@ options(help_type = "html")
    .rs.scalar(title)         
 })
 
+.rs.addJsonRpcHandler("get_vignette_description", function(topic, package)
+{
+   description <- tryCatch(
+      {
+         v <- vignette(topic, package)
+
+         Dir <- v$Dir
+         File <- v$File
+         if (grepl("[.]Rmd$", File))
+         {
+            description <- rmarkdown::yaml_front_matter(file.path(Dir, "doc", File))$description
```

Comment:
Do we have something that would extract the yaml front matter without using `rmarkdown` ?